scipy/special/add_newdocs.py

```diff
@@ -7467,11 +7467,62 @@ def add_newdoc(name, doc):
 
 add_newdoc("rgamma",
     """
-    rgamma(z)
+    rgamma(z, out=None)
 
-    Gamma function inverted
+    Reciprocal of the Gamma function.
+
+    Defined as :math:`1 / \Gamma(z)`, where :math:`\Gamma` is the
```

Comment:
looks like the `\G` is the problem & it gets caught by `pycodestyle` as well as full test suite for Python 3.6+. Not sure if it is better to try a "double escape" or if a docstring prefixed with `r` could work here